### PR TITLE
DSD-637: Removing alignText from the Notification component

### PR DESCRIPTION
**Pull request by EdwinGuzman**
Fixes JIRA ticket [DSD-637](https://jira.nypl.org/browse/DSD-637)

## This PR does the following:
- Removes the `alignText` prop from the `Notification` component. Storybook displays this prop so it seems it's available to use, but it's actually not. Leaving it can create confusion about what it should be doing when toggling its Control doesn't affect the UI.

## How has this been tested?

By checking that `alignText` was removed in the Controls tab.

### Checklist:

- [ ] I have updated the Storybook documentation accordingly.
- [ ] All new and existing tests passed.

### Front End Review:

- [ ] View [the example in Storybook]()

### CODE REVIEW

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr789-txvpmgewvazrk5fxpdggculsvy524fod.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61aa6fcbb880b2487411af11

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr789-txvpmgewvazrk5fxpdggculsvy524fod.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61aa6fcbb880b2487411af11

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr789-txvpmgewvazrk5fxpdggculsvy524fod.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61aa6fcbb880b2487411af11

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr789-txvpmgewvazrk5fxpdggculsvy524fod.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61aa6fcbb880b2487411af11

File: src/components/Notification/Notification.tsx [hunk 1]
```diff
@@ -22,7 +22,10 @@ interface BaseProps {
   notificationType?: NotificationTypes;
 }
 
-export interface NotificationProps extends BaseProps {
+type BasePropsWithoutAlignText = Omit<BaseProps, "alignText">;
```
Comment (bigfishdesign13) [inline src/components/Notification/Notification.tsx:26]: If the intent of this update is to remove the `alignText` prop, why is this or any other reference to `alignText` still in here?

**Review by bigfishdesign13 — COMMENTED** [review]
I think `alignText` was part of the initial development and, as you noticed, it was not used in anything.  So yes, it should be removed.  However, please see my one comment.

File: src/components/Notification/Notification.tsx (hunk 1 above)
Comment (EdwinGuzman) [reply src/components/Notification/Notification.tsx:26]: The actual `Notification` component doesn't use `alignText` but the `NotificationContent` component does use it. So it's only for internal use.

Comment (bigfishdesign13) [reply src/components/Notification/Notification.tsx:26]: Got it.  Thanks.

**Review by bigfishdesign13 — APPROVED** [review]
Wonderful.

**Comment by EdwinGuzman** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr789-txvpmgewvazrk5fxpdggculsvy524fod.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/61aa6fcbb880b2487411af11

**Pull request merged**